app/ts/components/requirements.tsx

```diff
@@ -94,11 +94,36 @@ export const Immutable = ( { checkBoxesArray } : { checkBoxesArray: OptionalSign
 	if (checkBoxesArray.deepValue === undefined) return <></>
 	const checkBoxes = checkBoxesArray.deepValue[checkBoxesArray.deepValue.length - 1]
 	if (checkBoxes === undefined || checkBoxes.type !== 'finalChild') return <></>
+
+	function humanReadableDateDelta(secondsDiff: number) {
+		const secondsInHour = 3600
+		const secondsInDay = secondsInHour * 24
+		const secondsInMonth = secondsInDay * 30
+		const secondsInYear = secondsInDay * 365
+
+		const years = Math.floor(secondsDiff / secondsInYear)
+		secondsDiff %= secondsInYear
+
+		const months = Math.floor(secondsDiff / secondsInMonth)
+		secondsDiff %= secondsInMonth
+
+		const days = Math.floor(secondsDiff / secondsInDay)
+
+		const parts: string[] = []
+		if (years > 0) parts.push(`${ years } years`)
+		if (months > 0) parts.push(`${ months } months`)
+		if (days > 0) parts.push(`${ days } days`)
+
+		return parts.length > 0 ? parts.join(' ') : 'less than a day'
```

Comment:
I feel like we should only show years and months if years > 0.  Maybe even only show years if it is greater than 5.

"100 years, 3 months, 7 days" is a bit silly compared to just "about 100 years".

Reply:
I think its meaningful information if its "1 years 2 months", but yeah, maybe not show them if years >5

Reply:
I made it so that months are only shown if years is under or equal to 5 and days only show if years is under or equal to 2